pkg/utils/db.go

```diff
@@ -53,7 +53,23 @@ func GetFlavor(ctx context.Context, db *sql.DB) (string, error) {
 }
 
 // GetAllServerID gets all slave server id and master server id
-func GetAllServerID(ctx context.Context, db *sql.DB) (map[int64]interface{}, error) {
+func GetAllServerID(ctx context.Context, db *sql.DB) (map[uint32]interface{}, error) {
+	serverIDs, err := GetSlaveServerID(ctx, db)
+	if err != nil {
+		return nil, err
+	}
+
+	masterServerID, err := GetServerID(db)
+	if err != nil {
+		return nil, err
+	}
+
+	serverIDs[masterServerID] = struct{}{}
+	return serverIDs, nil
+}
+
+// GetSlaveServerID gets all slave server id
+func GetSlaveServerID(ctx context.Context, db *sql.DB) (map[uint32]interface{}, error) {
```

Comment:
return a `map[uint32]interface{}` is easier to use, but it seems strange. :sweat_smile:

how about `map[uint32]struct{}`? (more like a `set` in other languages to hold nothing for the key, if using `interface{}`, it seems to want to hole anything).